### DISCUSSION TITLE
Dinamic Props Responsive

### DISCUSSION THREAD

**Discussion by IvanWala** [quasarframework/quasar#12565] [Ideas / Proposals]
I don't know if this possible in this moment. But can be cool add a feature liki this one:

https://vue3-mq.info/component/props.html

Specilally the md+, md-, md,... values.

I see this. 

https://quasar.dev/options/screen-plugin

But nothing similar to vue3-mq.

**Comment by metalsadman** [comment]
https://quasar.dev/style/visibility#window-width-related

**Reply by IvanWala** [reply]
Thanks @metalsadman  that's quite interesting but that only hides the element. Not renders or not depending on media queries. But i can try on this

**Reply by rstoenescu** [reply]
v-if="$q.screen..."

**Reply by IvanWala** [reply]
Specilally the **md+, md-,** md,... values.

**Reply by rstoenescu** [reply]
$q.screen.gt... $q.screen.lt.. (gt greater than; lt lower than)

**Reply by IvanWala** [reply]
Thank you @rstoenescu
